prisma/schema.prisma

```diff
@@ -156,12 +156,64 @@ model ComicIssue {
   collectionNft        CollectionNft?
   pages                ComicPage[]
   wallets              WalletComicIssue[]
+  collaborator         Collaborator[]
 
   @@unique([number, comicSlug])
   @@unique([slug, comicSlug])
   @@unique([title, comicSlug])
 }
 
+enum CollaboratorRole {
+  Writer
+  Artist
+  Colorist
+  Editor
+  Letterer
+  CoverArtist
+}
+
+model Collaborator {
+  id                Int         @id @default(autoincrement())
+  role              CollaboratorRole
+  name              String
+  comicIssueId      Int?
+  comicIssue        ComicIssue? @relation(fields: [comicIssueId], references: [id])
+  comicSlug         String?
+  comic             Comic?      @relation(fields: [comicSlug], references: [slug])
+  featuredCover     FeaturedCover?
+  stateFullCover    StateFullCover?         
+}
```

Comment:
Sharing this table is an anti-pattern I believe.
We will never have a row which has both a comicIssueId and a comicSlug, it will only have one of those foreign keys at any time. featuredCover and statefulCover will only have values if a specific role is related to a comic issue.

It makes sense to split these two tables and kill the nullable fields.